plugin/pkg/auth/authorizer/node/node_authorizer_test.go

```diff
@@ -104,6 +104,21 @@ func TestAuthorizer(t *testing.T) {
 			attrs:  authorizer.AttributesRecord{User: node0, ResourceRequest: true, Verb: "get", Resource: "secrets", Name: "secret0-pod0-node0", Namespace: "ns0"},
 			expect: authorizer.DecisionAllow,
 		},
+		{
+			name:   "list allowed secret via pod",
```

Comment:
add cases to make sure a list and watch without a name is not allowed

Reply:
done